core/src/main/resources/org/apache/spark/ui/static/historypage-template.html

```diff
@@ -20,17 +20,17 @@
   <thead>
     <tr>
       <th>
-        <span data-toggle="tooltip" data-placement="right" title="ID of the application.">
+        <span data-toggle="tooltip" data-placement="right" title="ID of this application.">
           App ID
         </span>
       </th>
       <th>
-        <span data-toggle="tooltip" data-placement="above" title="Name of the application.">
+        <span data-toggle="tooltip" data-placement="right" title="Name of this application.">
```

Comment:
Hello, you mean, the application is changed to this application? Or 'above' to 'right'?

first, consistent with other field titles.
![11](https://cloud.githubusercontent.com/assets/26266482/26663856/350b369e-46c0-11e7-8a0d-b44a7a9f3a80.png)

second, The tooltip placement to "above", the prompt position will always be displayed on the APP ID, rather than on the current title.